tools/c7n_tencentcloud/c7n_tencentcloud/resources/cvm.py

```diff
@@ -17,5 +25,81 @@ class resource_type(ResourceTypeInfo):
         version = "2017-03-12"
         enum_spec = ("DescribeInstances", "Response.InstanceSet[]", {})
         metrics_instance_id_name = "InstanceId"
-        paging_def = {"method": PageMethod.Offset, "limit": {"Key": "Limit", "value": 20}}
+        paging_def = {"method": PageMethod.Offset, "limit": {"key": "Limit", "value": 20}}
         resource_preifx = "instance"
+        taggable = True
+        batch_size = 10
+
+
+class CvmAction(TencentCloudBaseAction):
+    schema_alias = True
+
+    """cvm base api_method_name """
+
+    def process(self, resources):
+        for batch in chunks(resources, self.resource_type.batch_size):
+            params = self.get_request_params(batch)
+            if params is not None:
+                self.do_request(params)
+
+    def do_request(self, params):
+        """process_cvm"""
+        try:
+            client = self.get_client()
+            resp = client.execute_query(self.t_api_method_name, params)
+            failed_resources = jmespath.search("Response.Error", resp)
+            if failed_resources is not None:
+                raise PolicyExecutionError(f"{self.data.get('type')} error")
+            self.log.debug("%s resources: %s, cvm: %s",
+                           self.data.get('type'),
+                           params['InstanceIds'],
+                           params)
+        except (RetryError, TencentCloudSDKException) as err:
```

Comment:
its worth having a look at error handling more generally. 

generally a custodian policy which has an action error, will try and process the rest of the resources in the set before giving erroring to stop policy execution. its also important to note that several exceptions may not be meaningful in the context of the action and can be safely ignored, ie going to stop or delete an instance that is already terminated, or deleting a tag that doesn't exist, etc. the behavior wrt to that error handlng can vary if the api operation is for a single resource or for a batch, for the former simply continuing with the next resource, if the later if its ignorable/retryable is to remove that error'd resource from the set and try with the remainder. this behavior options here does depend on the behavior of the underlying api in terms of how it reports errors , and the state machine for the resource on the control plane, and the objective of the action in that context.